collections/_article/godot-foundation-update-2023.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Godot Foundation Update 2023"
+excerpt: "This year has marked significant milestones for the Godot Foundation, and it's all thanks to the incredible support from our community."
+categories: ["news"]
+author: Godot Foundation
+image: /storage/blog/covers/godot-foundation-update-2023.jpg
+date: 2023-12-05 12:00:00
+---
+
+This has been a monumental year for the Godot Foundation. We [announced](https://godotengine.org/article/godots-graduation-godot-moves-to-a-new-foundation/) the Foundation one year ago and have been busy since getting all the legalities  in place and transferring over all our assets from the SFC.
```

Comment:
```suggestion
This has been a monumental year for the Godot Foundation. We [announced](https://godotengine.org/article/godots-graduation-godot-moves-to-a-new-foundation/) the Foundation one year ago and have been busy since getting all the legalities in place and transferring over all our assets from the [Software Freedom Conservancy](https://sfconservancy.org/).
```